apps/src/lab2/types.ts

```diff
@@ -81,7 +81,7 @@ export interface Level {
   isAquaticLevel: boolean;
 }
 
-export interface LevelProperties {
+export interface ServerLevelProperties {
```

Comment:
Hmm, this is interesting...trying to think if makes sense to have this as a separate type, or just use the `LevelProperties` type throughout. An advantage with this is that we can narrow the boolean values to actual boolean types, but i'm trying to think if it's worth maintaining two different types for that. @molly-moen curious what you think?

Reply:
looks like you removed this in favor of the validator, that seems like the right call!